projects/ngx-d3timeline/src/lib/content/selectors/activity.selectors.ts

```diff
@@ -46,8 +53,39 @@ const selectDraggedToSeries = createSelector(
   valueToSeries
 );
 
+const selectDeltaTime = createSelector(
```

Comment:
`selectTimeDeltaForDrag` + see other renaming suggestions of `getDeltaTime` further down.. the two should be consistent.

Reply:
done